CIRC-5012 - Add IRONdb 0.19.4 Changelog (#56)
circonus/docs: content/irondb/release-notes.md

# Release Notes

## Changes in 0.19.4

2019-04-27

>  * Fix a bug in parsing FlatBuffers for raw data

Reviewer: ```suggestion
 * Fix a bug in parsing FlatBuffers for raw data.
```

>  * Fix null pointer exception crash on absent metric locator during /find.

Reviewer: ```suggestion
 * Fix null pointer exception crash on absent metric locator during `/find`.
```

>  * Improved performance of metric search indices improving performance of
>    initial start-time and tag searches where the category has wildcards
>    (e.g. `and(version-*:v1.*)`)

Reviewer: ```suggestion
 * Improved performance of metric search indices, reducing initial start-time
    and speeding up tag searches where the category has wildcards
   (e.g. `and(version-*:v1.*)`)
```

>  * Implement a variety of text-based /fetch transforms for counts and rates

Reviewer: ```suggestion
 * Implement a variety of text-based `/fetch` transforms for counts and rates.
```

>  * lua: Fix a bug where partially-initialized lua extensions were kept around

Reviewer: ```suggestion
 * lua: Fix a bug where partially-initialized lua extensions were kept around.
```

>  * CAQL: Add package `filter:*`

Reviewer: ```suggestion
 * CAQL: Add package `filter:*`.
```

>  * CAQL: Validate uuid passed to `metric:*` functions

Reviewer: ```suggestion
 * CAQL: Validate uuid passed to `metric:*` functions.
```
